### PR TITLE
refactor(gh-pages): add `new` badge

### PR DESCRIPTION
**Pull request by Natalie-Smirnova**
add `new` badge
image
image

### CODE REVIEW

File: pages/views/core/esl-mixin-element.njk [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: content
 title: ESL Mixin Element
 seoTitle: ESL Smart Library base mixin definition
 name: ESL Mixin Element
-tags: [core, beta]
+tags: [core, beta, new]
```
Comment (ala-n) [inline pages/views/core/esl-mixin-element.njk:6]:
```suggestion
tags: [core, new]
```

**Review by ala-n — APPROVED** [review]

File: pages/src/common/badge.less [hunk 2]
```diff
@@ -34,6 +34,11 @@
     background-color: #dc3545;
   }
 
+  &-success {
+    color: #212529;
+    background-color: #04c250;
+  }
```
Comment (NastaLeo) [inline pages/src/common/badge.less:40]: Maybe it makes sense to create variables with all these colors in pages\src\common\colors.less

**Review by NastaLeo — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Pull request merged**